app/models/manageiq/providers/vmware/infra_manager/vim_connect_mixin.rb

```diff
@@ -2,6 +2,11 @@ module ManageIQ::Providers::Vmware::InfraManager::VimConnectMixin
   extend ActiveSupport::Concern
 
   def connect(options = {})
+    service = options[:service] || 'vim'
+    send("#{service}_connect", options)
+  end
```

Comment:
Hm I feel like this should live in `Vmware::InfraManager`, seems weird that the vim_connect_mixin would call out to `cis_connect`

We'll just need to also add a `connect` to `Host` which is the other user of `VimConnectMixin`